### PR TITLE
mariadb-101: fix missing quotations in Makefile

### PR DESCRIPTION
**Pull request by dbh201**

### CODE REVIEW

**Comment by AndWac** [comment]
Please squash your commits and get rid of the superfluous commit messages.

**Comment by AndWac** [comment]
This PR caontains also 2 foreign commits.

**Comment by alarcher** [comment]
@dbh201 Use git pull --rebase origin oi/hipster to rebase your work on current main branch.

**Comment by dbh201** [comment]
Thanks @alarcher, that worked.

File: components/database/mariadb-101/Makefile [hunk 1]
```diff
@@ -46,8 +46,8 @@ CMAKE_PLUGINDIR.64 =	lib/$(MACH64)/plugin
 
 CMAKE_OPTIONS+= -DCMAKE_C_COMPILER=$(CC)
 CMAKE_OPTIONS+= -DCMAKE_CXX_COMPILER=$(CXX)
-CMAKE_OPTIONS+= -DCMAKE_C_FLAGS=$(CFLAGS)
-CMAKE_OPTIONS+= -DCMAKE_CXX_FLAGS=$(CFLAGS)
+CMAKE_OPTIONS+= -DCMAKE_C_FLAGS="$(CFLAGS) "
+CMAKE_OPTIONS+= -DCMAKE_CXX_FLAGS="$(CFLAGS) "
 
 CMAKE_OPTIONS+= -DINSTALL_LAYOUT=SVR4
```
Comment (AndWac) [inline components/database/mariadb-101/Makefile:52]: Just a note: When you change something in a package without increasing its VERSION_NUMBER you'll need to bump its COMPONENT_REVISION. We have now a special situtation as I changed the RELEASE_MAJOR recently. So changing COMPONENT_REVISION is optional one time.

**Pull request merged**